stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/MessageListListenerContainerImpl.kt

```diff
@@ -1,4 +1,4 @@
-package io.getstream.chat.android.ui.message.list.adapter.internal
+package io.getstream.chat.android.ui.message.list.adapter
```

Comment:
Had to move the impl class to the same package as the interface to comply with the requirements for sealed types.

Reply:
Maybe we can remove this interface?